projects/showcase/src/app/components/application-frame/application-frame-dialog/showcase-application-frame-dialog.component.html

```diff
@@ -1,10 +1,10 @@
 <systelab-app-frame #appFrame [userName]="userName" [userFullName]="userFullName" [title]="title" [menu]="menu" [tabs]="sidetabs"
     [actions]="sideactions" [menuBars]="menuBars"  [largeSideBar]="false" [logoIcon]="logoIcon" (selected)="doSelect($event)">
     <div header-content>
-        <button type="button" class="btn btn-sm">Button Top</button>
+        <button type="button" class="btn btn-outline-primary btn-sm">Button Top</button>
```

Comment:
This change implies to add "btn-outline-primary" in all the buttons

Reply:
This is a *DRAF* Pull request. It is not something final.